### ISSUE TITLE
New updates to API-M CICD process using APICTL

### ISSUE THREAD

**Issue by nilminiwso2** [wso2/docs-apim#4016]
Document the changes to the new API-M CICD process (using APICTL) that will be introduced.
Awaiting development.

**Comment by Mariangela** [comment]
@wasuradananjith , is this still applicable?

**Comment by Mariangela** [comment]
This has been labelled as a bug as it seems like there is missing content that is important.

**Comment by wasuradananjith** [comment]
> @wasuradananjith , is this still applicable?

Yes. There is a related issue [1] that has been already added to the RRT queue as well. 

[1] https://github.com/wso2/product-apim-tooling/issues/698
